PurchasesTests/Mocks/MockSKProduct.swift

```diff
@@ -2,6 +2,7 @@
 // Created by RevenueCat on 3/2/20.
 // Copyright (c) 2020 Purchases. All rights reserved.
 //
+import StoreKit
```

Comment:
I'm actually not sure this line is necessary. I was getting some build issues on Xcode 13 beta 2, but I think they were just beta issues.

Reply:
maybe we can add a todo to check again?